src/Sulu/Bundle/AdminBundle/Resources/public/js/components/header/main.js

```diff
@@ -498,10 +509,16 @@ define([], function() {
                     this.options.tabsOption,
                     {el: $container},
                     tabItem.componentOptions);
+
                 this.sandbox.start([{
                     name: tabItem.component,
                     options: options
-                }]);
+                }]).then(function(tabComponent) {
+                    // render title if view title is set and tab-option hideTitle title is false or not set
+                    if (!!this.options.title && (!tabComponent.tabOptions || !tabComponent.tabOptions.hideTitle)) {
```

Comment:
@nnatter can we rename 'hideTitle' to 'noTitle'? As a tab doesn't care about if the title gets hidden or not rendered, it just wants no title